runtime/vm/ContinuationHelpers.cpp

```diff
@@ -147,6 +147,29 @@ yieldContinuation(J9VMThread *currentThread)
 	return result;
 }
 
+void
+freeContinuation(J9VMThread *currentThread, j9object_t continuationObject)
+{
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, continuationObject);
+	if (NULL != continuation) {
```

Comment:
is it possible for this to be NULL

Reply:
if this continuation is freed after termination, GC can also call free when the continuation object is no longer referenced via the VMHelper wrapper.